README.md

```diff
@@ -5,3 +5,19 @@ Rendera
 A raytracing engine written in C++17.
 
 This project was made as a part of the Seasons of Code (SoC) initiative by the WnCC Club at IITB.
+
+>Note: The project uses some libraries which are added as submodules. To clone this project and build locally,
+
+
+```bash
+    git clone --recurse-submodules https://github.com/wermos/Rendera.git
+```
+
+To build the project and run locally,
+
+```bash
+    mkdir build
```

Comment:
If you pass in `-Bbuild` to CMake, then you don't need this line either.

Reply:
done!